<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1021</width>
     <height>581</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>729</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_password">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>设置不超过15位的密码</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_username">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>test1</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_password1">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_username">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_email">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_email">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>输入正确格式的邮箱以.net或者.com 结尾</string>
        </property>
        <property name="text">
         <string>2473837059@qq.com</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_password">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_password1">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>731</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_username_login">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_username_login">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>ttt</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_password_login">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_password_login">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>111</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
